### ISSUE TITLE
Setup/Configure/Install on Windows for a n00b

### ISSUE THREAD

**Issue by h-crypto-z** [cmditch/elm-ethereum-generator#20]
Hiya

I am trying to get the generator running on windows, but I am facing the following hurdle:
`C:\Source\elm-ethereum-generator-master>setup configure
Configuring elm-ethereum-generator-4.0.0...
setup: Encountered missing or private dependencies:
aeson -any,
ansi-wl-pprint -any,
cryptonite -any,
ilist -any,
optparse-applicative -any`

I have zero experience with haskell and need some help getting past this point, please?

->h_crypto_z

**Comment by h-crypto-z** [comment]
Seems I got it working.

With haskell installed (How to install haskell: http://www.haskell.org/platform/windows.html), I "accidentally" ran `cabal build`. This installed a bunch of required dependencies.

Thereafter I ran the cmd `cabal install`, but got the error `cabal.exe: filepath wildcard 'ChangeLog.md' does not match any files.`
I manually created the _changelog.md_ file and then ran the cmd `cabal install` again. This time it succeeded.

`C:\Source\elm-ethereum-generator-master>cabal install
Wrote tarball sdist to
C:\Source\elm-ethereum-generator-master\dist-newstyle\sdist\elm-ethereum-generator-4.0.0.tar.gz
Resolving dependencies...
Build profile: -w ghc-8.10.1 -O1
In order, the following will be built (use -v for more details):
 - elm-ethereum-generator-4.0.0 (lib) (requires build)
 - elm-ethereum-generator-4.0.0 (exe:elm-ethereum-generator-exe) (requires build)
Starting     elm-ethereum-generator-4.0.0 (lib)
Building     elm-ethereum-generator-4.0.0 (lib)
Installing   elm-ethereum-generator-4.0.0 (lib)
Completed    elm-ethereum-generator-4.0.0 (lib)
Starting     elm-ethereum-generator-4.0.0 (exe:elm-ethereum-generator-exe)
Building     elm-ethereum-generator-4.0.0 (exe:elm-ethereum-generator-exe)
Installing   elm-ethereum-generator-4.0.0 (exe:elm-ethereum-generator-exe)
Completed    elm-ethereum-generator-4.0.0 (exe:elm-ethereum-generator-exe)
Symlinking 'elm-ethereum-generator-exe.exe'
cabal.exe: Symlinking feature not available on Windows`

Symlinking failed... Where do I now find the executable...
